feature/profile/src/main/java/com/iteneum/profile/data/ProfileViewModel.kt

```diff
@@ -0,0 +1,95 @@
+package com.iteneum.profile.data
+
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.setValue
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.iteneum.Profile
+import com.iteneum.network.DataState
+import dagger.hilt.android.lifecycle.HiltViewModel
+import kotlinx.coroutines.launch
+import javax.inject.Inject
+
+/**
+ * File that contains [ProfileViewModel] that works as ViewModel for the Profile View.
+ * It will handle the data work & process when loading & be done with the view.
+ *
+ * @author Jose Miguel Garcia Reyes
+ */
+
+@HiltViewModel
+class ProfileViewModel @Inject constructor(): ViewModel() {
+    var profileModelMutable: Profile by mutableStateOf(Profile())
+        private set
+
+    fun getInformation() = viewModelScope.launch {
+        when (response) {
+            is DataState.Success -> {
+                profileModelMutable = response.data
+            }
+            is DataState.Error -> {
+                profileModelMutable = Profile(
+                    profileName = "Error",
+                    profileApartment = "Error",
+                    profilePhoneNumber = "1111111111",
+                    profileEmail = "Error",
+                    profileAddress = "Error"
+                )
+            }
+            is DataState.Loading -> {
+                profileModelMutable = Profile(
```

Comment:
this shouldn't be here, let's delete and put a "TODO"

Reply:
Got it. Modified!